src/Lunr/Vortex/APNS/ApnsPHP/APNSResponse.php

```diff
@@ -103,14 +103,14 @@ protected function set_statuses(array $endpoints, array $errors): void
 
                 switch ($status_code)
```

Comment:
Why not use the tryFrom method here too?

Reply:
Because it's two different enums that I'm checking here.